### DISCUSSION TITLE
Custom Landing Page -> Hidden Login/Admin area

### DISCUSSION THREAD

**Discussion by damanic** [btcpayserver/btcpayserver#2846] [Ideas / Feature Requests]
Can we have an option to set a custom landing page for BTCPayServer? 

I'm sure many hosts would prefer to show a custom HTML page to inquisitive visitors instead of exposing a login screen to the administration area.

It would also help to be able to shift the login/admin area onto a custom URL prefix. Eg. myserver.com/btcps_admin/login

This way the host can advertise whatever they like on the root domain, and expose access to the btcps user/admin area selectively.

**Comment by bolatovumar** [comment]
You can sort of do this already by creating a POS app, removing all products from it an including your custom HTML in the "Description" field for the POS app. You can then point your root domain to this app from "Customization Settings" section of the "Policies" page under "/server/policies". You can also include your custom CSS on the POS app edit page.

**Comment by damanic** [comment]
That's one way to hack something together as is,  we could probably find a couple more.  

A new APP that enables a custom home page would be one way to option this feature.  Although if this is the kind of feature that most hosts would use,  it might be a worthy addition to BTCPS customisation settings.

**Comment by dennisreimann** [comment]
That's a very good use case for a plugin.
